src/Compilers/CSharp/Portable/Symbols/Source/ParameterHelpers.cs

```diff
@@ -598,45 +605,66 @@ static void addERR_ParamsCantBeWithModifier(BindingDiagnosticBag diagnostics, Sy
                 // which reports the error on the type following "this".
 
                 // error CS1100: Method '{0}' has a parameter modifier 'this' which is not on the first parameter
-                diagnostics.Add(ErrorCode.ERR_BadThisParam, thisKeyword.GetLocation(), owner.Name);
+                diagnostics.Add(ErrorCode.ERR_BadThisParam, thisKeyword.GetLocation(), owner?.Name ?? "");
             }
-            else if (parameter.IsParams && owner.IsOperator())
+            else if (isParams && owner != null && owner.IsOperator())
```

Comment:
>owner != null [](http://example.com/codeflow?start=33&length=13)

We typically use reference equality when checking a `Symbol` for `null`. Consider using `owner is { }` or `(object)owner != null`.